pkgs/by-name/zu/zutty/package.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchgit,
+  pkg-config,
+  freetype,
+  wafHook,
+  python3,
+  libXmu,
+  glew,
+  ucs-fonts,
+  nixosTests,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "zutty";
+  version = "0.16-1-g04b2ca3";
```

```suggestion
  version = "0.16-unstable-2024-08-18";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning